### PR TITLE
[14.0][MIG] membership_prorate_variable_period: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT32111

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @pedrobaeza @chienandalu

**Review by chienandalu — APPROVED** [review]
Thanks!

File: membership_prorate_variable_period/tests/test_membership_prorate_variable_period.py [hunk 1]
```diff
@@ -20,220 +21,110 @@ def setUp(self):
...
-            {
-                "account_id": account,
-                "product_id": self.product.id,
-                "price_unit": self.product.list_price,
-                "name": "Membership prorate fixed",
-                "invoice_id": invoice.id,
-                "quantity": 1.0,
-            }
-        )
+        self.product.membership_date_from = date(2017, 1, 1)
```
Comment (pedrobaeza) [inline membership_prorate_variable_period/tests/test_membership_prorate_variable_period.py]: These changes are not needed. Odoo accepts strings as well.

**Review by pedrobaeza — COMMENTED** [review]
No need to change date strings into dates

File: membership_prorate_variable_period/tests/test_membership_prorate_variable_period.py (hunk 1 above)
Comment (ernestotejeda) [reply membership_prorate_variable_period/tests/test_membership_prorate_variable_period.py:66]: Should I change it as it was before or should I keep the change?

Comment (ernestotejeda) [reply membership_prorate_variable_period/tests/test_membership_prorate_variable_period.py]: @pedrobaeza change done. I reverted that change and others as well to reduce the diffs to make it easier to review

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**